src/utils.ts

```diff
@@ -999,7 +1061,7 @@ export async function getNonCompliantApprovalAddress(erc721Contract: Web3.Contra
     // CRYPTOKITTIES check
     promisifyCall<string>(c => erc721Contract.kittyIndexToApproved.call(tokenId, c), onError),
     // Etherbots check
-    promisifyCall<string>(c => erc721Contract.partIndexToApproved.call(tokenId, c), onError),
+    // promisifyCall<string>(c => erc721Contract.partIndexToApproved.call(tokenId, c), onError),
```

Comment:
@alexanderatallah I commented this out since I did not understand it's usage as if you look in the tests it passes a reference of the Cryptokittes contract instances to this method; `erc721Contract` is the CK contract. `erc721Contract.partIndexToApproved` should fail because `partIndexToApproved` is on the Etherbots contract AFAIK.

Please let me know if these assumptions are true so that I can then remove the commented out line

Reply:
@taylorjdawson I think we need to keep this and fix the onError handler, otherwise I could write an etherbots test that would fail (this is currently failing on a cryptokitties test, but there is no etherbots test).

Also, the line `let approvedAddr = await promisifyCall<string>(c => erc721.getApproved.call(tokenId, c))` in `public async approveNonFungibleToken` would also have problems

Reply:
Let's just remove onError and make that argument undefined. don't think it's needed - I just wasn't getting errors with geth and added it to make sure we throw if we haven't updated the erc721 abi to include weird approval methods, but we have done that